src/xrc/xmlres.cpp

```diff
@@ -1701,9 +1701,9 @@ static wxColour GetSystemColour(const wxString& name)
         SYSCLR(wxSYS_COLOUR_BTNTEXT)
         SYSCLR(wxSYS_COLOUR_INACTIVECAPTIONTEXT)
         SYSCLR(wxSYS_COLOUR_BTNHIGHLIGHT)
-        SYSCLR(wxSYS_COLOUR_BTNHILIGHT)
+        SYSCLR(wxSYS_COLOUR_BTNHIGHLIGHT)
```

Comment:
If I'm reading this diff correctly, there are now 2 wxSYS_COLOUR_BTNHIGHLIGHT items.

Dave
 


      From: VZ <notifications@github.com>
 To: wxWidgets/wxWidgets <wxWidgets@noreply.github.com> 
 Sent: Tuesday, June 30, 2015 2:49 PM
 Subject: Re: [wxWidgets] Cleaning (#46)
   
In src/xrc/xmlres.cpp:> @@ -1701,9 +1701,9 @@ static wxColour GetSystemColour(const wxString& name)
>          SYSCLR(wxSYS_COLOUR_BTNTEXT)
>          SYSCLR(wxSYS_COLOUR_INACTIVECAPTIONTEXT)
>          SYSCLR(wxSYS_COLOUR_BTNHIGHLIGHT)
> -        SYSCLR(wxSYS_COLOUR_BTNHILIGHT)
> +        SYSCLR(wxSYS_COLOUR_BTNHIGHLIGHT)
These names are part of the library interface too and changing them would break the existing XRC files using them, we should continue to accept them (because there is really no good reason not to).—
Reply to this email directly or view it on GitHub.